commerce_coordinator/apps/commercetools/catalog_info/edx_utils.py

```diff
@@ -48,33 +47,21 @@ def get_edx_lms_user_name(customer: CTCustomer):
     return customer.custom.fields[EdXFieldNames.LMS_USER_NAME]
 
 
-def get_edx_successful_stripe_payment(order: CTOrder) -> Union[CTPayment, None]:
+def get_edx_successful_payment_info(order: CTOrder):
     for pr in order.payment_info.payments:
         pmt = pr.obj
-        if pmt.payment_status.interface_code == PAYMENT_STATUS_INTERFACE_CODE_SUCCEEDED \
-            and pmt.payment_method_info.payment_interface == EDX_STRIPE_PAYMENT_INTERFACE_NAME and \
-                pmt.interface_id:
-            return pmt
-    return None
+        if pmt.payment_status.interface_code == PAYMENT_STATUS_INTERFACE_CODE_SUCCEEDED and pmt.interface_id:
+            return pmt, pmt.payment_method_info.payment_interface
+    return None, None
 
 
 def get_edx_payment_intent_id(order: CTOrder) -> Union[str, None]:
```

Comment:
this can be either stripe payment intent id or paypal order ID
in frontend we call this psp_payment_id, for consistency this should be same
or if we want to follow CT name (like ct_transaction_interaction_id)
this should be called ct_payment_provider_id

Reply:
Do you mean to update the function name or payment class property here?